urdf/psen_scan.urdf.xacro

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://ros.org/wiki/xacro">
+  <xacro:macro name="psen_scan" params="prefix">
+    <xacro:property name="min_range" value="0.1" /> <!-- TODO correct dimensions -->
```

Comment:
why do dimensions have to be corrected?

Reply:
Come to think of it, we don't need them any more ;-)